src/ui/server/content-strings/pages/insurance/eligibility/index.ts

```diff
@@ -130,6 +130,17 @@ const ELIGIBLE_TO_APPLY_ONLINE = {
   PAGE_TITLE: "You're eligible to apply for insurance",
   INSET: 'This does not automatically guarantee cover.',
   BODY: 'You now need to fill in the application so we can assess the risks around your exports and your buyer.',
+  PRIVACY: {
+    INTRO: "We're committed to protecting the privacy and security of the personal information we hold. Read our",
+    PRIVACY_NOTICE: {
+      LINK: {
+        TEXT: 'privacy notice',
+        HREF: LINKS.EXTERNAL.PRIVACY_NOTICE,
+      },
+      TEXT: 'for more details.',
+    },
+  },
+  CONTINUE_SUBMIT: 'Once you continue your application you will have 30 days to submit it.',
```

Comment:
Can we use a constant for 30, maybe `SUBMISSION_DEADLINE_IN_DAYS`, like `SUBMISSION_DEADLINE_IN_MONTHS`.

Would be nice to also find and replace the other few instances of "30 days" with the new constant

Reply:
done